### PR TITLE
SASS configuration

### PR DESCRIPTION
**Pull request by Thom-IT**
**What does this PR do?**
it setup the Syntactically Awesome Style Sheets(SASS) for react
**Description of Task to be completed**
-setup button.css
-setup fancy-button.scss
**How should this be manually tested?**
git clone https://github.com/atlp-rwanda/wolves-bn-frontend.git
run npm install
run npm run dev to run the application and access the base scss
**What are the relevant Pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/#174772168

### CODE REVIEW

**Review by Cherubln — CHANGES_REQUESTED** [review]
Nice @Thom-IT 
I saw there are some development stuffs in the webpack.config.js like isdevelopment, devServer, etc . It will be much better if anything related to development  is kept in the webapack.dev.js because webpack.config.js is only for production.

File: src/styles/scss/main.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    @media(max-width:$mobile) {
+        @content;
+        
+    }
+    
+}
+.main{
+   // @include flexCenter;  //use mixins(remove parameter from flexCenter or use it or functions, or variables below
+    @include flexCenter(row); //use row or column
+    // display: flex;
```
Comment (minega25) [inline src/styles/scss/main.scss]: Lets remove all commented code.

File: webpack.config.js [hunk 2]
```diff
@@ -29,6 +39,52 @@ module.exports = {
...
+      {
+        test: /\.module\.s(a|c)ss$/,
+        loader: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              modules: true,
+              sourceMap: isDevelopment
+            }
```
Comment (minega25) [inline webpack.config.js]: I thought we got rid of the isDevelopment variable

File: webpack.dev.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 /* eslint-disable import/no-extraneous-dependencies */
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 
+const isDevelopment = process.env.NODE_ENV !== 'production';
```
Comment (minega25) [inline webpack.dev.js:7]: There is no need for this variable anymore

**Review by minega25 — CHANGES_REQUESTED** [review]

**Review by minega25 — APPROVED** [review]

**Review by Cherubln — APPROVED** [review]
Looks good to me

**Pull request merged**